templates/AureliaSpa/ClientApp/boot.ts

```diff
@@ -0,0 +1,15 @@
+import {Aurelia} from 'aurelia-framework';
+import 'bootstrap/dist/css/bootstrap.css';
+import 'bootstrap';
+ 
+// comment out if you don't want a Promise polyfill (remove also from webpack.config.js)
+import * as Bluebird from  'bluebird';
+Bluebird.config({ warnings: false });
+
+export function configure(aurelia: Aurelia) {
+  aurelia.use
+    .standardConfiguration()
+    .developmentLogging();
```

Comment:
Looks like I'll add a check for localhost in window location to decide whether to enable logging.  Couldn't find a better suggestion not relying on a gulp task to actually edit code (that's what I've used in past).  Only happens when app is loading so doesn't seem too bad.